imperative/python/megengine/functional/elemwise.py

```diff
@@ -132,7 +132,36 @@ def sub(x, y):
 
 
 def mul(x, y):
-    r"""Element-wise `multiplication`."""
+    r"""Returns the tensor representing the element-wise multiplication of the two input tensors(also known as Hadamard product).
+
+    Args:
+        x: input tensor.
+        y: input tensor.
```

Comment:
```suggestion
        y: second input tensor. Must be compatible with `x` (see :ref:`broadcasting-rule` ). Should have a numeric data type.
```

这里的 :ref:`broadcasting-rule` 在生成文档时会指向：

https://megengine.org.cn/doc/stable/zh/user-guide/model-development/functional/manipulation.html#broadcasting-rule